src/Kart/Leaflet.js

```diff
@@ -114,11 +114,22 @@ class Leaflet extends React.Component {
     this.props.handleLokalitetUpdate(lng, lat, this.map.getZoom());
   };
 
-  handleInfobox = (bool) => {
+  handleInfobox = bool => {
     this.setState({ showInfobox: bool });
   };
 
-  handleClick = (e) => {
+  removeTabIndexes = () => {
+    // Remove default leaflet link in map so it
```

Comment:
igjen, vil den ikke nå bli helt utilgjengelig for brukere uten tastatur?
enig i at det er praktisk for oss, men tror kanskje vi bør la dem være en eller annen plass i hierarkiet?

Reply:
Kan ta det bort så det blir tilgjengelig igjen.